packages/neo-one-website/src/components/reference/TypeFilter.tsx

```diff
@@ -0,0 +1,36 @@
+// tslint:disable no-any
+import * as React from 'react';
+import { styled } from 'reakit';
+import { Select } from './Select';
+import { TypeFilterOptions } from './types';
+
+const REFERENCE_TYPE_VALUES: ReadonlyArray<TypeFilterOptions> = [
```

Comment:
I would move this to where `TypeFilterOptions` is declared, otherwise they could easily get out of sync.